### PR TITLE
DEV: rename the "upgrade" route to "update" and change references.

### PR DESCRIPTION
**Pull request by vinothkannans**
This PR will rename the route "`/admin/upgrade`" to "`/admin/update`" and it will add a redirect for the old route.

Follow-up to the PR: https://github.com/discourse/docker_manager/pull/208

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
+    get "/update/:id" => "admin#index"
     get "/docker/repos" => "admin#repos"
     get "/docker/latest" => "admin#latest"
     get "/docker/progress" => "admin#progress"
     get "/docker/ps" => "admin#ps"
 
-    post "/docker/upgrade" => "admin#upgrade"
-    delete "/docker/upgrade" => "admin#reset_upgrade"
+    post "/docker/update" => "admin#update"
+    delete "/docker/update" => "admin#reset_update"
```
Comment (janzenisaac) [inline config/routes.rb:15]: I don't see updates to the controller actions, do we need to do that as well?

Comment (vinothkannans) [reply config/routes.rb]: Good catch! I now reverted the changes in route methods. I'm not going rename the method names in this PR (only the routes).

File: lib/docker_manager/upgrader.rb [hunk 2]
```diff
@@ -257,9 +257,9 @@ def status(val)
     publish("status", val)
   end
 
-  def log_version_upgrade
+  def log_version_update
```
Comment (janzenisaac) [inline lib/docker_manager/upgrader.rb:260]: We renamed the method, but I don’t see another instance of the method in the list of changes. Do we have more places to update?

Comment (vinothkannans) [reply lib/docker_manager/upgrader.rb]: I didn't intent to update that method name too. I fixed it now. I doubled checked all other method names again.

**Review by janzenisaac — APPROVED** [review]

**Pull request merged**